ark/utils/deepcell_service_utils_test.py

```diff
@@ -41,14 +47,35 @@ def test_create_deepcell_output(mocker):
         assert os.path.exists(os.path.join(output_dir, 'fov1_feature_0.tif'))
         assert os.path.exists(os.path.join(output_dir, 'fov2_feature_0.tif'))
 
-        # if fovs is None, all .tif files in input dir should be taken
+        # fovs.zip file should be removed from input folder after completion
+        assert not os.path.isfile(os.path.join(input_dir, 'fovs.zip'))
+
+        os.remove(os.path.join(output_dir, 'fov1_feature_0.tif'))
+        os.remove(os.path.join(output_dir, 'fov2_feature_0.tif'))
+
+        # test with mixed fov/file list
+        create_deepcell_output(deepcell_input_dir=input_dir, deepcell_output_dir=output_dir,
```

Comment:
Can you add one image that has .tiff as well? We want to catch the error Adam pointed out